.github/actions/package-review/action.yml

```diff
@@ -0,0 +1,29 @@
+# Relevant documentation:
```

Comment:
What's the difference between an "action" and a "workflow"?

Reply:
https://docs.github.com/en/actions/getting-started-with-github-actions/core-concepts-for-github-actions#action